pm/recipient.go

```diff
@@ -43,6 +43,9 @@ type Recipient interface {
 
 	// TxCostMultiplier returns the multiplier -
 	TxCostMultiplier(sender ethcommon.Address) (*big.Rat, error)
+
+	// MinEV returns the recipients EV requirement as configured on startup
+	MinEV() *big.Int
```

Comment:
Should this be called `EV()` instead? The EV configured on node start up is the fixed value required for tickets. It just happens to also be the value for O's minimum credit requirement.

Could we also return a `big.Rat` here (we'd have to convert the config value from a `big.Int`) so that callers don't need to worry about the conversion?